woocommerce-services.php

```diff
@@ -623,8 +624,10 @@ protected function get_service_object_by_id( $class_name, $service_id ) {
 		 * @return mixed
 		 */
 		public function woocommerce_shipping_methods( $shipping_methods ) {
-
-			$shipping_service_ids = $this->get_service_schemas_store()->get_all_service_ids_of_type( 'shipping' );
+			$shipping_service_ids = $this->get_service_schemas_store()->get_all_shipping_method_ids();
+			if ( ! $shipping_service_ids ) {
+				return $shipping_methods;
+			}
```

Comment:
See: https://github.com/Automattic/woocommerce-services/pull/890/files#r102328474 (applies to all below as well)